.github/workflows/ci.yml

```diff
@@ -33,8 +33,8 @@ jobs:
       - name: Build source
         run: spago build --no-install --purs-args '--censor-lib --strict'
 
-#      - name: Run tests
-#        run: spago test --no-install
+      - name: Run tests
+        run: spago build --no-install --then "node -e \"import { main } from './output/Test.Main/index.js'; main();\" --input-type=module"
```

Comment:
This isn't needed because we run `npx pulp test` in the next step.

We had to disable `spago test` because it doesn't yet work on ES modules.

Reply:
Sure enough, sorry for the oversight!

Reply:
No worries!